lang/terraform/terraform.talon

```diff
@@ -0,0 +1,28 @@
+mode: user.code_terraform
+mode: command
```

Comment:
Not quite correct... these consecutive `mode` conditions are OR'd together, so this file is now always active when in `command` mode or when a .tf file is open in any mode. Take a look at some of the other language files (or in #603).

Reply:
Oh damn! Again I oriented myself by looking at the java language files. I'm going to submit a pull request soon to fix the issues found here which are also present in the java language files. I hope it's correct now.
Also I had made another error where I used the wrong mode (`user.code_terraform` instead of `user.terraform`). Good thing you pointed me in the right direction :grimacing:

Reply:
Man we really gotta implement https://github.com/knausj85/knausj_talon/issues/603#issuecomment-980106400.  This language mode stuff is getting pretty krufty 😅